app/_src/kubernetes-ingress-controller/guides/services/tcp.md

```diff
@@ -249,9 +252,12 @@ echo-plaintext   192.0.2.3   3m18s
 
 ## Route based on SNI
 
+{:.note}
+> For Gateway API, we use TLSRoute for SNI-based routing because it's specifically designed to handle TLS traffic with SNI information, while still proxying the underlying TCP stream. This is still considered TCP proxying, just with the added capability of routing based on the SNI in the TLS handshake.
```

Comment:
```suggestion
> For Gateway API, we use `TLSRoute` for SNI-based routing because it's specifically designed to handle TLS traffic with SNI information, while still proxying the underlying TCP stream. This is still considered TCP proxying, just with the added capability of routing based on the SNI in the TLS handshake.
```